css/responsive/mobile.styl

```diff
@@ -75,12 +74,14 @@ body.single-collection
     margin: 0
 
 .single-collection-row
+  display: block
+  text-align: center
   justify-content: center
 
 .single-collection-item
   flex: 0 1 auto
-  max-height: 100%
-  width: auto
+  max-height: none
```

Comment:
is this just to allow a scroll if needed?

Reply:
if there is max height the items will squish on landscape